tendermint/src/vote.rs

```diff
@@ -101,7 +101,9 @@ impl lite::Vote for SignedVote {
     }
 
     fn sign_bytes(&self) -> Vec<u8> {
-        AminoMessage::bytes_vec(&self.vote.to_owned())
+        let mut buf = Vec::with_capacity(self.vote.encoded_len() + 4);
```

Comment:
4 bytes should be correct but it would still be cleaner to take the length and its encoded length. Prost provides a method for that. As far as I remember it was called `encoded_len_varint`. Let me double check.